lib/sensors/low_power_current.cpp

```diff
@@ -0,0 +1,39 @@
+#include <low_power_current.hpp>
+
+namespace hyped::sensors {
+
+std::optional<LowPowerCurrent> LowPowerCurrent::create(core::ILogger &logger,
+                                                       std::shared_ptr<io::II2c> i2c,
+                                                       const std::uint8_t channel
+                                                       = kDefaultI2cAddress)
+{
+  return LowPowerCurrent(logger, i2c, channel);
+}
```

Comment:
if this has no failure state then there is no need for the create function

Reply:
if there is configuration required that is yet to be implemented then ignore me

Reply:
^^ TODOLater